corehq/apps/hqadmin/templates/hqadmin/partials/reset-pillow-checkpoint-modal.html

```diff
@@ -8,7 +8,8 @@
         <div class="modal-body">
             <p>{% trans "Are you sure you want to reset checkpoint for this pillow?" %}</p>
             <div class="alert">
-            <p><i class="icon-warning-sign"></i><strong>{% trans "You still have to RESTART the pillow manually!" %}</strong></p>
+            <p><i class="icon-warning-sign"></i> {% trans "You still have to RESTART the pillow manually!" %}</p>
+            <p><i class="icon-warning-sign"></i> {% trans "NOTE: this may invoke a huge indexing task that could slow down many parts of the site!" %}</p>
```

Comment:
Pillows don't generally cause indexing except for the MVP ones but we're in the process of addressing those. Fine to leave the message though.